docs/presets/README.md

```diff
@@ -3,91 +3,76 @@
 Here is a collection of community-submitted configuration presets for Starship.
 If you have a preset to share, please [submit a PR](https://github.com/starship/starship/edit/master/docs/presets/README.md) updating this file! 😊
 
-## Nerd Font Symbols
+## Default configuration with Nerd Font Symbols
 
 This preset doesn't change anything except for the symbols used for each module.
 If emojis aren't your thing, this might catch your eye!
```

Comment:
Btw: this sentence is ambiguous: i never figured out if this means the normal default need a nerdfont or the normal default work with a normal font (my guess, as i never installed a nerd font and it works).

Reply:
@jankatins, most of the modules use emojis by default and the are not handled by the font. The above configuration is not using any emojis, but Nerd Font symbols instead.

The configuration contains a lot more settings, than mere colors & symbols. What phrasing would you suggest to clarify that only symbols and colors are changed in the first two configuration presets?